### ISSUE TITLE
Cannot import an obj sequence

### ISSUE THREAD

**Issue by zakattack321** [neverhood311/Stop-motion-OBJ#42]
I assume this is a user error? If so, what am I doin wrong? I included a private vimeo link of what the issue is

https://vimeo.com/user49972809/review/404857198/8cfb845b4b

**Comment by neverhood311** [comment]
@zakattack321 I can't access that Vimeo link. It says "Sorry, we couldn’t find that page"

**Comment by StephenMcAvoy** [comment]
I can't access that link, either.

**Comment by zakattack321** [comment]
https://vimeo.com/404857198

Try this

**Comment by neverhood311** [comment]
It looks like the error is happening inside the Blender .OBJ importer. Is it possible to send me one of the .OBJ files in the sequence?

**Comment by zakattack321** [comment]
Here's the full sequence

https://drive.google.com/open?id=13sPETh4CTTlOnlFbk7QUfDo8vPdiEGkm

**Comment by StephenMcAvoy** [comment]
I downloaded the first OBJ in the sequence and Blender refused to import it with the standard addon. Importing it to another editor (PoseRay) failed as well. PoseRay reported that the model had no faces. Which Program did you use to make the sequence?

**Comment by neverhood311** [comment]
@zakattack321 Thanks for sharing that sequence. When I tried to import a single one of them (not using Stop motion OBJ) into Blender, I got a similar error. So I opened up the file in Notepad and it looks like these files aren't OBJ files at all. How did you create these files?
After a quick Google search for "PGEOMETRY V5" (the first line in each of these "obj" files) I found [this forum post](https://www.sidefx.com/forum/topic/52170/), which may be helpful to you.
